run.py

```diff
@@ -1,30 +1,36 @@
-from config.path import ukb_raw_fn
+from config.path import ukb_loader_dir, valid_ids_path, ukb_pfile_path
 from config.pca_config import pca_config
 from config.qc_config import qc_config
-from config.split_config import split_config
-from preprocess.load import ukb_into_plink_binary
+from config.split_config import iid_split_config, non_iid_split_config
+from preprocess.central_qc import central_qc
 from preprocess.pca import PCA
 from preprocess.qc import QC
 from preprocess.split import SplitNonIID, SplitIID
+from utils.plink import run_plink
 
-if __name__ == '__main__':
-    # runs the whole pipeline
+import logging
+import sys
 
-    # convert data into PLINK format
-    prefix_raw = ukb_into_plink_binary(ukb_raw_fn)
 
-    # QC and PCA for central dataset
-    prefix_qc = QC.qc(input_prefix=prefix_raw, qc_config=qc_config)
-    PCA.run(input_prefix=prefix_qc, pca_config=pca_config, output_tag='smth')
+if __name__ == '__main__':
+    # runs the whole pipeline
+    logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
```

Comment:
thanks for adding a logger, could you also add time output?
such as 
logging.basicConfig(
    format='%(asctime)s %(levelname)-8s %(message)s',
    level=logging.INFO,
    datefmt='%Y-%m-%d %H:%M:%S')